src/languageServerInstaller.ts

```diff
@@ -104,9 +100,7 @@ export class LanguageServerInstaller {
 }
 
 async function getLsVersion(dirPath: string): Promise<string> {
-	const lsBinPath = `${dirPath}/terraform-ls`;
-
-	fs.accessSync(lsBinPath, fs.constants.X_OK);
+	fs.accessSync(lsBinPath(dirPath), fs.constants.X_OK);
```

Comment:
Unfortunately it looks like the method references the `lsBinPath` const in several locations; you'll need to assign it to a variable.

We'll need a new variable name though; maybe `binPath`?
```suggestion
	const binPath = lsBinPath(dirPath);

	fs.accessSync(binPath, fs.constants.X_OK);
```

(I couldn't fix the others; GitHub wouldn't let me add notes on those lines.)

Reply:
Good catch!